examples/ex15.cpp

```diff
@@ -19,6 +19,14 @@
 //               ex15 -m ../data/square-disc.mesh
 //               ex15 -m ../data/escher.mesh -r 2 -tf 0.3
 //
+//               Kelly estimator:
+//
+//               ex15 -est 1
```

Comment:
This runs a lot faster than the ZZ estimator. There seems to be about an order of magnitude difference in the errors being predicted.

I found that the command `./ex15 -est 1 -e 0.0001` gave more similar results

Reply:
The sample run has been updated.